lib/metasploit/framework/password_crackers/hashcat/formatter.rb

```diff
@@ -89,6 +89,10 @@ def hash_to_hashcat(cred)
     when /^mscash2$/
       # hash-mode: 2100
       return cred.private.data.split(':').first
+    when /netntlm(v2)?/
+      #            netntlm, netntlmv2
+      # hash-mode: 5500     5600
+      return cred.private.data
```

Comment:
Hashcat uses the same format as John The Ripper for netntlm and netntlmv2 hashes. See their example wiki page for more information: https://hashcat.net/wiki/doku.php?id=example_hashes.